docs/en/engines/table-engines/integrations/s3.md

```diff
@@ -174,6 +174,10 @@ Received exception from server (version 23.4.1):
 Code: 48. DB::Exception: Received from localhost:9000. DB::Exception: Reading from a partitioned S3 storage is not implemented yet. (NOT_IMPLEMENTED)
 ```
 
+## Inserting data
+
+Note that rows can only be inserted into new files. There are no merge cycles or file split operations. Once a file is written, subsequent inserts will fail. See more details [here](/integrations/s3#inserting-data).
```

Comment:
Let's also add a note about  `s3_truncate_on_insert` and `s3_create_new_file_on_insert` (same as in RU version). Also, maybe we can unify this, since we have those settings described under `#settings`, thoughts?

Reply:
Done